extensions/figma-files/src/components/FileListItem.tsx

```diff
@@ -10,29 +10,35 @@ import { OpenPageSubmenuAction } from "./OpenPageSubmenuAction";
 TimeAgo.addDefaultLocale(en);
 const timeAgo = new TimeAgo("en-US");
 
-export default function FileListItem(props: { file: File; extraKey?: string; onVisit: (file: File) => void }) {
-  const { file, extraKey, onVisit } = props;
+export default function FileListItem(props: {
+  file: File;
+  extraKey?: string;
+  onVisit: (file: File) => void;
+  showingDetail: boolean;
+}) {
+  const { file, extraKey, onVisit, showingDetail } = props;
 
-  const accessoryTitle = String(timeAgo.format(new Date(file.last_modified)));
+  const dateUpdated = String(timeAgo.format(new Date(file.last_modified)));
   const fileIdentifier = extraKey ? `${file.key}-${extraKey}` : file.key;
   return (
     <List.Item
       id={fileIdentifier}
       title={file.name}
       icon={file.thumbnail_url}
-      accessoryTitle={accessoryTitle}
+      accessories={[{ text: showingDetail ? "" : dateUpdated }]}
       actions={
         <ActionPanel>
           <ActionPanel.Section>
             <OpenProjectFileAction file={props.file} onVisit={onVisit} />
-            <CopyToClipboardAction content={`https://figma.com/file/${file.key}`} />
+            <Action.CopyToClipboard content={`https://figma.com/file/${file.key}`} />
           </ActionPanel.Section>
           <ActionPanel.Section>
             <OpenPageSubmenuAction file={props.file} onVisit={onVisit} />
           </ActionPanel.Section>
           <DevelopmentActionSection />
         </ActionPanel>
       }
+      detail={<List.Item.Detail markdown={`![Thumbnail](${file.thumbnail_url}) \n Last updated: ${dateUpdated}`} />}
```

Comment:
How about showing the `Last updated` information and maybe other details as `List.Item.Detail.Metadata`? This would look more beautiful and align with other extensions better.

You can find more info [here](https://developers.raycast.com/api-reference/user-interface/list#list.item.detail.metadata).